### ISSUE TITLE
Warning de Github lorsqu'on publish les gh-pages pour le repo ToulouseJS

### ISSUE THREAD

**Issue by MoOx** [francejs/francejs.github.com#112]
Nous utilisons toulousejs.francejs.org et (ça fait longtemps qu'on avait pas mis à jour), j'ai reçu une alerte au build.

Qui gère les enregistrements DNS? Le nécessaire peut-il être fait pour éviter de recevoir l'alerte à chaque build?

Merci

---

```
The page build completed successfully, but returned the following warning:

Your site's DNS settings are using a custom subdomain, toulousejs.francejs.org, that's set up as an A record. We recommend you change this to a CNAME record pointing at francejs.github.io. Instructions on configuring a subdomain for use with GitHub Pages can be found at: https://help.github.com/articles/setting-up-a-custom-domain-with-github-pages/ 

For information on troubleshooting Jekyll see:

 https://help.github.com/articles/using-jekyll-with-pages#troubleshooting

If you have any questions you can contact us by replying to this email.
```

**Comment by kud** [comment]
Il me semble que c'est @naholyr

**Comment by nfroidure** [comment]
On a eu le même souci chez ChtiJS mais il me semblait que c'était résolu.

**Comment by MoOx** [comment]
Bump car c'est pénible.
T'as fait quelque chose @nfroidure ?

**Comment by nfroidure** [comment]
Nope, j'ai confondu avec un autre warning (sur les ips car elles avaient changé).

Mais ce warning on l'a aussi chez ChtiJS, il ont encore changé de méthode (c'est plus sécure comme ça).

C'est tjrs @naholyr qui a le contrôle je pense. Ping :)

**Comment by nfroidure** [comment]
PS: @naholyr si tu peux aussi ajouter le CNAME pour ChtijsJS :angel:

**Comment by MoOx** [comment]
Et sinon on peut faire un truc pour ça?
On peut aider? On peut avoir accès au registar pour mettre ça à jour?
Je recois un mail à chaque update du site, et des fois ca peut vite être chiant...

**Comment by naholyr** [comment]
vu, je pensais qu'on avait déjà partagé les droits d'admin technique, je regarderai demain soir je suis en déplacement

**Comment by naholyr** [comment]
Avant

```
*                       28800  A      192.30.252.153
*                       28800  A      192.30.252.154
@                       28800  A      192.30.252.153
@                       28800  A      192.30.252.154
```

Après

```
*                       28800  CNAME  francejs.github.io.
@                       28800  CNAME  francejs.github.io.
```

Prise en compte dans quelques heures. Je renouvelle dans la foulée (expiration prévue mars 2016) ;)

**Comment by MoOx** [comment]
Pour info t'es chez qui pour avoir le support du CNAME?

**Comment by naholyr** [comment]
Bookmyname mais tu l'as partout ça *a priori*, je passe par contre toujours par les interfaces dites "expert", c'est à dire le simple textarea ;)

**Comment by nfroidure** [comment]
@naholyr tu peux aussi ajouter ce sous-domaines pour une orga à part ? :):
chtijs                       28800  CNAME  chtijs.github.io.

**Comment by naholyr** [comment]
@nfroidure done, effectivement il faut des entrées spécifiques du coup
